### PR TITLE
Products page

### PR DESCRIPTION
**Pull request by aayushjain-code**

### CODE REVIEW

File: pages/Product Page/products.css [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+	display: grid;
+	grid-template-columns: 1fr 3fr;
+}
+
+
+.filter-container {
+	margin-top: 10rem;
+	margin-bottom: 10rem;
+	margin-left: 2rem;
+	background-color: var(--bg-color-1);
```
Comment (mikesch803) [inline pages/Product Page/products.css:12]: CSS variable naming is not good

**Review by mikesch803 — CHANGES_REQUESTED** [review]
your UI looks good, the text over the image has a background color which is not looking is good, you can remove the background color

File: components/checkbox.css [hunk 2]
```diff
@@ -0,0 +1,67 @@
+/* The container-checkbox */
+.container-checkbox {
+	display: block;
+	position: relative;
+	padding-left: 35px;
+	margin-bottom: 12px;
```
Comment (MuskanChhatrasal) [inline components/checkbox.css:6]: Try to use relative units.
Else UI looks good.

**Review by MuskanChhatrasal — APPROVED** [review]

File: pages/Product Page/products.css (hunk 1 above)
Comment (aayushjain-code) [reply pages/Product Page/products.css:12]: Will try to make it better thanks for pointing

**Pull request merged**